resources/monitoring/values.yaml

```diff
@@ -40,6 +40,14 @@ global:
   imagePullSecrets: []
   # - name: "image-pull-secret"
 
+resourceSelector:
```

Comment:
this value by default should be empty, configuration should be applied only for production profile via config overrides